capstone/src/main/webapp/group.html

```diff
@@ -72,11 +72,12 @@ <h4 class="align-vertical">Members</h4>
           <span id="add-close" onclick="addModalListeners('add-member-modal', 'add-close')">&times;</span>
           <h2>Add Group Member</h2>
           <form id="add-member" onsubmit="addMember()" method="POST">
-            <label for="member-email-input">Enter an email</label>
+            <label for="member-email-input">Enter email</label>
             <input type="text" class="input-field" name="email-input" id="email-input">
             <p id=memberUpdate></p>
             <button class="add-member-btn">Add</button>
           </form>
+          <button class="add-member-btn" onclick="loadTestData()">Load Test Data</button>
```

Comment:
I think one thing to note is that pressing this button would overwrite Anika's data (Users w/ longitude and latitude values) if hers is already loaded in the db. An alternative way of handling it could be to have your NameTestData servlet `load-on-startup` (specified in the `web.xml` file). [This link](https://beginnersbook.com/2017/07/load-on-startup-tag-in-web-xml-file-with-example/) goes into how you can specify the order of the servlets, so as long as yours comes before Anika's, there shouldn't be an issue with her data being overwritten.

This will also help avoid the case of someone accidentally pressing the button twice and loading duplicate data (wouldn't actually be duplicate thanks to the user ids, would just be overwritten, but servlet would still go through the unnecessary work of putting each user in the datastore).